tools/static-code-analysis/pmd/suppressions.properties

```diff
@@ -3,3 +3,8 @@ org.eclipse.smarthome.io.console.karaf.internal.OSGiConsole=SystemPrintln
 org.eclipse.smarthome.io.console.rfc147.internal.CommandWrapper=SystemPrintln
 org.eclipse.smarthome.io.console.rfc147.internal.OSGiConsole=SystemPrintln
 org.eclipse.smarthome.core.internal.events.ThreadedEventHandler=EmptyIfStmt
+org.eclipse.smarthome.config.xml.ConfigDescriptionConverter=AvoidCatchingNPE
+org.eclipse.smarthome.io.rest.core.internal.thing.ThingResource=AvoidCatchingNPE
+org.eclipse.smarthome.test.java.JavaTest=AvoidCatchingNPE
```

Comment:
Perhaps okay for relaxed test writings, but hey, we are using pure Java tests and no Groovy, so should be easy to fix (you could also do it in a separate PR and create an tracker issue now if too much tests needs to be fixed).